package-lock.json

```diff
@@ -1,8 +1,1157 @@
 {
   "name": "selfsigned",
   "version": "1.10.12",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This project still seems to use `package-lock.json` version 1. Can you redo your changes with `npm` version 6 installed? This way the diff of `package-lock.json` should be much smaller.

Reply:
Oh sure :)

Reply:
My local node 6 version does not update the `package-lock.json`. Any ideas?

Reply:
You need to start with a v1 file of `package-lock.json`. I'm not sure if npm 6 will downgrade manually.

Reply:
> You need to start with a v1 file of `package-lock.json`. I'm not sure if npm 6 will downgrade manually.

I tried different things:

1. Removed the whole `package-lock.json` file -> File gets not created anymore
2. Manually replaced `lockfileVersion` by `1` -> Nothing happens

Before each try I removed the `node_modules` directory to start fresh.